### PR TITLE
ref: Switch to `workspace.dependencies`

### PR DESCRIPTION
**Pull request by Altair-Bueno**
Old PR got messed up due to some changes on main

- Old PR: https://github.com/jprochazk/garde/pull/27

### CODE REVIEW

**Comment by Altair-Bueno** [comment]
The PR should be ready to merge. I'll like to have this before merging `axum-garde` and a couple other fixes into the repo

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
-members = [
-    "garde_tests",
-    "garde_derive",
-    "garde",
-]
+members = ["garde", "garde_*"]
+
+[workspace.dependencies]
+# Workspace crates
+garde = { version = "0.10.0", path = "./garde" }
```
Comment (jprochazk) [inline Cargo.toml:6]: I use [cargo-workspaces](https://github.com/pksunkara/cargo-workspaces) to publish new releases, and it currently doesn't support workspace inheritance when bumping versions (https://github.com/pksunkara/cargo-workspaces/issues/82). Could you revert just `garde` and `garde_derive` back to using version+path directly in the individual crates' `Cargo.toml` files?

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,26 @@
 [workspace]
-members = [
-    "garde_tests",
-    "garde_derive",
-    "garde",
-]
+members = ["garde", "garde_*"]
```
Comment (jprochazk) [inline Cargo.toml:2]: I would prefer to have these explicitly listed out, especially if there aren't that many of them.

**Review by jprochazk — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (Altair-Bueno) [reply Cargo.toml:10]: Does it work now?

Comment (jprochazk) [reply Cargo.toml:10]: Yes, thanks!

**Review by jprochazk — APPROVED** [review]

**Comment by jprochazk** [comment]
Thanks! 🥇

**Pull request merged**